### DISCUSSION TITLE
Gets stuck loading/logging in

### DISCUSSION THREAD

**Discussion by JetForMe** [cryptoAlgorithm/Swiftcord#184] [Q&A]
I just installed Swiftcord via brew (0.6.1), entered my credentials, and now it's just spinning the Discord logo with a button saying "This is taking a while, want to logout?" I tried logging out and back in, but it just spiiiiinnnns.

**Comment by cryptoAlgorithm** [comment]
Hi, do give the latest release from GitHub releases (0.7.1) a shot. It has some fixes not present in 0.6.1 for this issue. However, some users still do encounter this issue on that release, so do keep us updated on whether that fixes the issue.
